tools/dynamatic/dynamatic.cpp

```diff
@@ -639,20 +640,31 @@ int main(int argc, char **argv) {
       return 1;
     }
 
-    // Read the script line-by-line and execute its commands
+    // Read the script line-by-line and execute its commands supported
+    // delimeters: '\n' and ';'
     std::string line;
-    while (std::getline(inputFile, line))
-      if (!line.empty() && !StringRef(line).starts_with("#")) {
-        llvm::outs() << PROMPT << line << "\n";
-        handleInput(line);
-      }
+    while (std::getline(inputFile, line, '\n')) {
+      if (StringRef(line).starts_with("#"))
```

Comment:
Looking at this I think I handle comments very badly (what if there is a `#` character in the middle of a line?). In any case, this is something I can fix after this PR.